.gitignore

```diff
@@ -156,6 +156,7 @@ ajax/libs/ng-pdfviewer/r0.2.1/
 ajax/libs/prettyCheckable/.2.1.1/
 ajax/libs/Hyphenator/HyphenatorBeta12/
 ajax/libs/Hyphenator/ersion_*
+ajax/libs/libphonenumber/libphonenumber-*/
```

Comment:
What is the reason why it will not update? All the new versions come with that prefix? If so, yes, I'll like to discuss with its author.